react-components/tsconfig.json

```diff
@@ -14,5 +14,5 @@
     "moduleResolution": "bundler",
     "allowSyntheticDefaultImports": true
   },
-  "include": ["src/*", "stories/*"]
+  "include": ["src/**/*", "stories/*"]
```

Comment:
Since src/* only includes files directly under the src directory. Technically it didn't matter much since we use webpack to bundle, but vscode / eslint doesn't like it, so changed it for convinience.